Adds a terms of service agreement on fresh installs of the extension

MaxReinisch opened the pull request:
Mark wanted a similar ToS agreement to the [Alexa Internet Extension](https://chrome.google.com/webstore/detail/alexa-traffic-rank/cknebhggccemgcnbidipinkifmmegdel?hl=en).  

It's functioning properly if on a fresh install, when you click the extension icon, it opens a new tab with a ToS agreement.  If you click agree, it sets the default popup back to index.html. Else, it just closes the window and doesn't change the browserAction behavior.  

The things I'd like to get feedback on:
- Any bugs?
- Style/Content of the ToS page
- Should I give any UI feedback for clicking agree or cancel? 
    - Right now, either option just closes the window.
    - The Alexa Internet extension redirects to a "success" page on agree.

vbanos commented on welcome.html:
```diff
@@ -0,0 +1,36 @@
+      <div id="content">
+        <h1>Welcome!</h1>
+        <p>Here’s your brand new Internet Archive browser extension.</p>
+        <p>The extension collects and stores information about the search queries you make and the web pages you view to provide you with information about those pages and to help Internet Archive measure the Web. In some cases, information collected by the extension may be personally identifiable, but we do not attempt to analyze any information collected by the extension to determine the identi ...
```
You say "measure the Web"

Maybe we wanted to say "archive the web"?

MaxReinisch replied:
"Help the Internet Archive archive the web" doesn't sound great.  Maybe I'll replace IA with WBM in this case

vbanos commented on welcome.html:
```diff
@@ -0,0 +1,36 @@
+  <head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Welcome!</title>
```
Welcome to the Internet Archive Wayback Machine extension!

MaxReinisch commented:
I also added a setting to un-agree to the ToS.  This is mostly for debugging purposes to show the welcome page next time the user clicks on the icon.  This setting should be removed and replaced with a setting to opt out of data collection while still letting them use the extension.
